### PR TITLE
[DGPFUS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by Otruke**
SHOW TABLES 1_1
show database1_1

### CODE REVIEW

**Review by Otruke — COMMENTED** [review]
database 1_3

File: submissions/rufusOtolorin/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (admin_id) REFERENCES admins (id),
+    FOREIGN KEY (food_id) REFERENCES food (id)
+);
+
+-- TO START CREATING CUSTOMERS ACCOUNT TABLE
+CREATE TABLE customers_account (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database/init.sql:41]: The admin_id should not be in this table.

File: submissions/rufusOtolorin/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    name VARCHAR(255) NOT NULL,
+    phone_number VARCHAR(100) NOT NULL,
+    address VARCHAR(255) NOT NULL,
+    payment_method VARCHAR(255),
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database/init.sql:45]: payment_method, etc as seen on the payment verification page. The first four columns mentioned are required and the other columns regarding payment are optional.
Change line 45 to:   payment_method ENUM('card', 'cash', 'transfer') NULL DEFAULT 'card'

File: submissions/rufusOtolorin/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    PRIMARY KEY (id),
+    FOREIGN KEY (admin_id) REFERENCES admins (id)
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database/init.sql:47]: This also is not needed in this table.

File: submissions/rufusOtolorin/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+);
+
+-- NOW TO MAKE THE ORDERS TABLE
+CREATE TABLE orders (
+	id INT NOT NULL AUTO_INCREMENT,
+    vendor_id INT NOT NULL,
+    customers_id INT ,
+    admin_id INT NOT NULL,
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database/init.sql:55]: Line 55 is not needed in this table.

File: submissions/rufusOtolorin/database/init.sql [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    PRIMARY KEY (id),
+    FOREIGN KEY (vendor_id) REFERENCES vendor_food (id),
+    FOREIGN KEY (admin_id) REFERENCES admins (id)
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database/init.sql:58]: Replace line 58 with FOREIGN KEY (customer_id) REFERENCES customer(id)

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @Otruke! Kindiy make the requested changes and revert when done. Thank you!

**Review by Mayowa2020 — APPROVED** [review]

**Pull request merged**